From: Takashi Iwai <tiwai@suse.de>
Subject: doc: Fix title markers

The title marker lines have to be aligned with the previous lines.
The error was caught by asciidoctor, which tends to be picker than
asciidoc.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 doc/obfs-local.asciidoc  |    2 +-
 doc/obfs-server.asciidoc |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/doc/obfs-local.asciidoc
+++ b/doc/obfs-local.asciidoc
@@ -1,5 +1,5 @@
 obfs-local(1)
-===========
+=============
 
 NAME
 ----
--- a/doc/obfs-server.asciidoc
+++ b/doc/obfs-server.asciidoc
@@ -1,5 +1,5 @@
 obfs-server(1)
-============
+==============
 
 NAME
 ----
